src/Parser.php

```diff
@@ -206,7 +206,7 @@ public function __construct()
         // ? provides for relaxed parsing of messages without trailing parameters properly demarcated
         $trailing = "(?: :?[^$null$crlf]*)";
         $params = "(?P<params>$trailing?|(?:$middle{0,14}$trailing))";
-        $name = "[$letter$number](?:[$letter$number\\-]*[$letter$number])?";
+        $name = "[$letter$number](?:[$letter$number:\/\\-]*[$letter$number])?";
```

Comment:
Escaped the backslash in cf4537b779f29cde02e883b77e1e13fce5f83ff9 until #34 is resolved.